patchwork-vanilla-patches/src/README.md

```diff
@@ -0,0 +1,8 @@
+# patchwork-vanilla-patches
+A collection of bugfixes and other changes to vanilla behavior that Forge makes.
+
+Bugfixes should be put here instead of other modules whenever possible, in order to force those fixes to be done in a way
```

Comment:
Generally I use softwrap for markdown, and don't manually wrap lines.

Reply:
156 columns seems awful long for editing in IDEA/etc, which lacks soft wrap.

Reply:
Either way, a one line wrap does not affect the actual displayed file.